common/lib/protocol-definitions/package.json

```diff
@@ -66,6 +66,14 @@
       },
       "InterfaceDeclaration_IClientConfiguration": {
         "backCompat": false
+      },
+      "InterfaceDeclaration_ISummaryAuthor": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "InterfaceDeclaration_ISummaryCommitter": {
```

Comment:
your suppression isn't quite right either. removed things get a modified name. look at the generated file for what to suppress